pkg/graph/resolvers/it_gov_task_statuses.go

```diff
@@ -89,8 +89,33 @@ func BizCaseDraftStatus(intake *models.SystemIntake) (models.ITGovDraftBusinessC
 }
 
 // GrtMeetingStatus calculates the ITGovGRTStatus for the GrtMeeting section for the system intake task list for the requester view
-func GrtMeetingStatus(intake *models.SystemIntake) models.ITGovGRTStatus {
-	return models.ITGGRTSCantStart
+func GrtMeetingStatus(intake *models.SystemIntake) (models.ITGovGRTStatus, error) {
+
```

Comment:
I did have it like you suggest actually, I just changed to use switch statements, because I thought it might be clearer and more aligned with the other status calculations ([here is a view of that approach ](https://github.com/CMSgov/easi-app/pull/2132/files/8d862fc48efd08e7ba51554e746dfc43294e0c4f))

Reply:
That link goes to this PR.

Reply:
Hmm, when I click it, it goes to the PR, but if you look, it selects the commits to view (which show the other style I previously had)

Reply:
Talked about this in pairing, going to be changed back to if-else.

Reply:
thanks Dylan, I changed it back, I decided to leave it as an early return instead of using an else block.